data/strings/types_strings.txt

```diff
@@ -18193,17 +18194,100 @@
     pl = Odkryte skały
```

Comment:
```
    nl = Kale rots
```
can't insert suggestion at the right place

Reply:
thanks!